sleap_roots/series.py

```diff
@@ -2,60 +2,102 @@
 
 import attrs
 import numpy as np
-from pathlib import Path
 import sleap_io as sio
-from typing import Optional, Tuple, List, Union
-
 import matplotlib
 import matplotlib.pyplot as plt
 import seaborn as sns
 
+from typing import Dict, Optional, Tuple, List, Union
+from attr import attrib
```

Comment:
Use `attrs` API, not the legacy `attr` one. [See these docs](https://www.attrs.org/en/stable/names.html).

Reply:
Done